codegen-sbt/src/main/scala/caliban/codegen/CompileTimeCalibanPlugin.scala

```diff
@@ -0,0 +1,223 @@
+package caliban.codegen
+
+import caliban.codegen.CalibanSourceGenerator.TrackedSettings
+import sbt.Keys._
+import sbt.{ Compile, Def, _ }
+
+import java.io.File
+
+/**
+ * User-oriented documentation:
+ * ----------------------------
+ *
+ * TODO Jules: Write doc
+ */
+object CompileTimeCalibanServerPlugin extends AutoPlugin {
+  override def requires = plugins.JvmPlugin
+  override def trigger  = noTrigger
+
+  override lazy val projectSettings: Seq[Def.Setting[_]] =
+    Seq(
+      libraryDependencies ++= Seq(
+        "com.github.ghostdogpr" %% "caliban-tools" % BuildInfo.version % Compile,
+        "dev.zio"               %% "zio"           % "1.0.11"          % Compile
```

Comment:
This is not needed

Reply:
Already brought by `caliban-tools`? 🤔

Reply:
Yeah caliban-tools depends on both client and core (server) and core depends on zio.